### PR TITLE
calculate Fibanachi and Factorial

### PR DESCRIPTION
**Pull request by b1zman**

### CODE REVIEW

**Comment by b1zman** [comment]
Beta

File: hw2_epam_lab/src/main/java/hw2/Fibonachi.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package hw2;
+
+import java.util.ArrayList;
+
+public class Fibonachi {
+
+    public static String сalculate(int loopType, int n) {
+        ArrayList<Integer> fibanachi = new ArrayList<>(n);
```
Comment (Jericho-d) [inline hw2_epam_lab/src/main/java/hw2/Fibonachi.java:8]: Don't use ArrayList to define new list

List<Integer> fibonacci= new ArrayList<>(n);

File: hw2_epam_lab/src/main/java/hw2/Fucktorial.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }
+        if (loopType == 3) {
+            for (int i = 0; i < n; i++) {
+                fucktorial = n * n;
+                i++;
+            }
+        } else if (loopType > 3 || loopType == 0) {
+            System.out.println("Хули ты вводишь, не видишь, что у нас мощности компухтера ограничены?");
+        }
+        return "ФАКториал "+ n + " = "+ fucktorial;
```
Comment (Jericho-d) [inline hw2_epam_lab/src/main/java/hw2/Fucktorial.java:30]: For one use you can choose String.format("ФАКториал %s = %s", n, fucktorial);
Don't forget that String.format() isn't a good idea for multiple call or string

File: hw2_epam_lab/src/main/java/hw2/Fibonacci.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                for (int i = 0; i < n; i++) {
+                    fibanachi.add(firstValue);
+                    thirdValue = firstValue + secondValue;
+                    secondValue = firstValue;
+                    firstValue = thirdValue;
+                    count++;
+                }
+            }
+
+            default -> throw new IllegalStateException("Error loopType: " + loopType);
```
Comment (kislyakdmitry) [inline hw2_epam_lab/src/main/java/hw2/Fibonacci.java:47]: Зачётно, молодец

**Pull request merged**